bundle/src/main/java/com/adobe/acs/commons/reports/models/ContainingPageReportCellCSVExporter.java

```diff
@@ -37,7 +39,7 @@
   public String getValue(Object obj) {
     Resource result = (Resource) obj;
     PageManager pageMgr = result.getResourceResolver().adaptTo(PageManager.class);
-    Page page = pageMgr.getContainingPage(result);
+    Page page = Optional.ofNullable(pageMgr).map(p -> p.getContainingPage(result)).orElse(null);
```

Comment:
Instead of using orElse on line 42 you could combine the next line as another map call and then to orElse(null) in the return statement